.github/workflows/sycl_linux_build_and_test.yml

```diff
@@ -110,6 +110,18 @@ jobs:
       run: |
         export XDG_CACHE_HOME=$GITHUB_WORKSPACE/os_cache
         cmake --build $GITHUB_WORKSPACE/build --target check-clang
+    - name: check-mlir-sycl
+      if: ${{ always() && !cancelled() && steps.build.outcome == 'success' }}
+      run: |
+        cmake --build $GITHUB_WORKSPACE/build --target check-mlir-sycl
+    - name: check-cgeist
+      if: ${{ always() && !cancelled() && steps.build.outcome == 'success' }}
+      run: |
+        cmake --build $GITHUB_WORKSPACE/build --target check-cgeist
+    - name: check-polygeist-opt
+      if: ${{ always() && !cancelled() && steps.build.outcome == 'success' }}
+      run: |
+        cmake --build $GITHUB_WORKSPACE/build --target check-polygeist-opt
     - name: check-sycl
```

Comment:
Don't you want to remove existing checks? I suppose they might be not relevant (at least for now).

Reply:
I think they would become relevant quickly enough

Reply:
Okay. We need to check that we have enough CI resources to handle extended pre-commit for another branch.